packages/mobile/src/import/saga.ts

```diff
@@ -32,41 +45,71 @@ import { assignAccountFromPrivateKey, waitWeb3LastBlock } from 'src/web3/saga'
 
 const TAG = 'import/saga'
 
+export const MAX_BALANCE_CHECK_TASKS = 5
+export const MNEMONIC_AUTOCORRECT_TIMEOUT = 5000 // ms
+
 export function* importBackupPhraseSaga({ phrase, useEmptyWallet }: ImportBackupPhraseAction) {
```

Comment:
How would you feel about adding a few analytics events here?
There are three cases that would be interesting to measure imo:
- Valid phrase input by user.
- Invalid phrase input but corrected successfully.
- Invalid phrase and we weren't able to correct it.

I don't have a specific use case for these metrics right now to be honest, I just have curiosity of how much this will help. My gut feeling says a lot, but would like to have confirmation on that.

Reply:
Good callout. I'll working on adding better analytics!

Reply:
I've added analytics in 24927ce